dev-vcs/git/git-2.45.2.recipe

```diff
@@ -133,12 +151,17 @@ REQUIRES_scalar="
 	"
 
 # The `subversion` package includes perl bindings that this tool needs.
-# Note: `subversion` still built with gcc2 on 32 bits (matching perl there, at least).
 REQUIRES_svn="
 	haiku$secondaryArchSuffix
 	git$secondaryArchSuffix == $portVersion base
+	git${secondaryArchSuffix}_perl == $portVersion
 	subversion
```

Comment:
I forgot adding `$secondaryArchSuffix` here after subversion was switched to secondaryArch. Will fix this when revbumping for the new perl.